packages/design-tokens/src/js/typography/fontFamilies.ts

```diff
@@ -1,3 +1,4 @@
 export const fontFamilies = {
-  euclidCircularB: 'Euclid Circular B',
+  centraNo1: 'Centra No1',
+  sans: 'Centra No1',
```

Comment:
Adding a new `sans` token to match CSS variables. It is not currently used so it may make sense to remove it